### DISCUSSION TITLE
Script Compilation Errors for Installing Old Gen RER

### DISCUSSION THREAD

**Discussion by ZukoIsBack** [Aelto/W3_RandomEncounters_Tweaks#194] [Q&A] [answered]
These are the errors I am getting. I do have w3eeredux, but I followed the manual merge instructions and had everything merge. I appear to have source errors though, and I'm not sure how to fix.

Error [mod_sharedutils_damagemodifiers]local\sharedutils\damagemodifiers\main.ws(10): 'sharedutils_damage_modifiers' is not a member of '&handle:CNewNPC'
Error [mod_sharedutils_damagemodifiers]local\sharedutils\damagemodifiers\main.ws(11): 'sharedutils_damage_modifiers' is not a member of '&handle:CNewNPC'
Error [mod_sharedutils_damagemodifiers]local\sharedutils\damagemodifiers\main.ws(19): 'sharedutils_damage_modifiers' is not a member of '&handle:CNewNPC'
Error [mod_sharedutils_damagemodifiers]local\sharedutils\damagemodifiers\main.ws(20): 'sharedutils_damage_modifiers' is not a member of '&handle:CNewNPC'
Error [mod_sharedutils_dialogchoices]local\sharedutils\dialog_choices.ws(46): 'isAcceptedChoiceAvailable' is not a member of '&handle:CR4HudModuleDialog'
Error [mod_sharedutils_dialogchoices]local\sharedutils\dialog_choices.ws(57): 'isAcceptedChoiceAvailable' is not a member of '&handle:CR4HudModuleDialog'
Error [mod_sharedutils_dialogchoices]local\sharedutils\dialog_choices.ws(68): 'lastAcceptedChoice' is not a member of '&handle:CR4HudModuleDialog'
Error [mod_sharedutils_dialogchoices]local\sharedutils\dialog_choices.ws(71): 'lastAcceptedChoice' is not a member of '&handle:CR4HudModuleDialog'
Error [mod_sharedutils_dialogchoices]local\sharedutils\dialog_choices.ws(75): 'isAcceptedChoiceAvailable' is not a member of '&handle:CR4HudModuleDialog'
Error [mod_sharedutils_dialogchoices]local\sharedutils\dialog_choices.ws(79): 'isAcceptedChoiceAvailable' is not a member of '&handle:CR4HudModuleDialog'
Error [mod_sharedutils_helpers]local\sharedutils\helpers\interactions.ws(48): Could not find function 'addInteractionEventListener'
Error [mod_sharedutils_mappins]local\sharedutils\mappins.ws(92): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_mappins]local\sharedutils\mappins.ws(143): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_mappins]local\sharedutils\mappins.ws(144): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_mappins]local\sharedutils\mappins.ws(150): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_mappins]local\sharedutils\mappins.ws(151): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_mappins]local\sharedutils\mappins.ws(156): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_mappins]local\sharedutils\mappins.ws(167): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_mappins]local\sharedutils\mappins.ws(168): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_mappins]local\sharedutils\mappins.ws(175): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_mappins]local\sharedutils\mappins.ws(176): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_mappins]local\sharedutils\mappins.ws(181): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_mappins]local\sharedutils\mappins.ws(188): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_mappins]local\sharedutils\mappins.ws(197): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_mappins]local\sharedutils\mappins.ws(198): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_mappins]local\sharedutils\mappins.ws(204): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_mappins]local\sharedutils\mappins.ws(205): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_mappins]local\sharedutils\mappins.ws(210): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_mappins]local\sharedutils\mappins.ws(264): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_mappins]local\sharedutils\mappins.ws(265): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_noticeboards]local\sharedutils\noticeboards\errand_injector.ws(131): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_noticeboards]local\sharedutils\noticeboards\errand_injector.ws(132): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_noticeboards]local\sharedutils\noticeboards\errand_injector.ws(138): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_noticeboards]local\sharedutils\noticeboards\errand_injector.ws(139): 'customMapPins' is not a member of 'handle:CR4Player'
Error [mod_sharedutils_noticeboards]local\sharedutils\noticeboards\errand_injector.ws(143): 'customMapPins' is not a member of 'handle:CR4Player'

**Comment by Aelto** [comment]
I doubt you merged anything, literally all of the errors you listed mean the RER code isn't merged or not included in the merge

**Reply by ZukoIsBack** [reply]
Well that's bizarre because I used script merger and everything automerged apart from the one issue with FHUD built into eeredux. It went from the left of script merger to the right and was listed in the merged files. I'll try a clean install and start from square one.

**Reply by ZukoIsBack** [reply]
I just did a clean install, and I have the same issue. Here is my list of merges
![Screenshot 2023-04-25 185905](https://user-images.githubusercontent.com/127753102/234446921-57206926-032e-43e1-807e-71ab7da45900.png)

**Reply by Aelto** [reply]
The guide assumes you've installed W3EER by overwriting EE's files. That means if you didn't overwrite them anytime EE is mentioned in the guide, it will be W3EER in your case. If you merge against W3EE but you didn't overwrite the its scripts with the redux ones, and if your EE has a priority there is a chance none of the merges will be loaded and it will continue to use W3EER scripts without the RER changes in them

**Reply by ZukoIsBack** [reply]
Hello Aelto, thank you for your reply. I did overwrite the scripts (cut the scripts from redux and pasted into content for w3ee), so there is not a w3eeredux file sitting unmerged for any of the above merges. I'm thinking it must have to do with my mod load order. I am at work right now, but I will post the load order when I get home. Thank you for your help.

**Reply by ZukoIsBack** [reply]
Sorry for the delay. Here is the load order and my scripts in general.
![scripts](https://user-images.githubusercontent.com/127753102/235013788-e56acbc8-3af9-4b92-ba0c-6858b7c34606.png)
![loadorder](https://user-images.githubusercontent.com/127753102/235013818-d61a544b-9f23-4ae3-84f3-c7bc8672cff0.png)

**Reply by ZukoIsBack** [reply, marked as answer]
I figured it out. The mod interactive poise finisher was screwing it up and I had to prioritize the merged files. Thank you for your help.
